pkgs/os-specific/linux/firmware/fwupd/default.nix

```diff
@@ -162,6 +163,12 @@ stdenv.mkDerivation rec {
       # needs a different set of modules than po/make-images
       inherit installedTestsPython;
     })
+
+    # Find the correct lds and crt name when specifying -Defi_ldsdir
+    (fetchpatch {
+      url = "https://github.com/fwupd/fwupd/commit/52cda3db9ca9ab4faf99310edf29df926a713b5c.patch";
+      sha256 = "sha256-JtRsMJ5/Twsq5GcYrt6BYU1UGu88jvF9PtpX9Vs6UkM=";
```

Comment:
This hash seems problematic (for me and borg, at least)-- replacing with traditional hash (I got `0hsj79dzamys7ryz33iwxwd58kb1h7gaw637whm0nkvzkqq6rm16`) fixed the issue.

-----

Otherwise, LGTM :).

Reply:
Hmm, that is going to be problematic if things do not support SRI hashes.

Reply:
Oh it is from `fechpatch` init, will fix that in a separate PR.

Reply:
Will be fixed by https://github.com/NixOS/nixpkgs/pull/79987

Reply:
For now, I switched back to `base32` hash.